### DISCUSSION TITLE
Welcome to learning-codeql Discussions!

### DISCUSSION THREAD

**Discussion by SummerSec** [SummerSec/learning-codeql#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yogurt-ui** [comment]
想请问一下,codeql-debug的项目，下载文件，本地运行py文件，按照讲解对应修改各路径：依旧出现讲解中类似是库未更新的报错（could not resolve type TaintTracking::Configuration），想问下这类问题具体应该如何调试解决
